README.md

```diff
@@ -27,9 +27,9 @@ This repository provides a modern, well-structured template for developing WordP
 
```

Comment:
I would say the readme.md, probably should get a complete wipe, and start with documentation for the plugin, as most of the stuff in the readme.md isn't really applicable if it is not a template